### PR TITLE
fix ts fe test error.

### PR DESCRIPTION
**Pull request by lanluo-nvidia**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: tests/modules/hub.py [hunk 1]
```diff
@@ -19,10 +19,66 @@
         "No GPU found. Please check if installed torch version is compatible with CUDA version"
     )
 
+if not importlib.util.find_spec("torchvision"):
+    print(f"torchvision is not installed, skip models download")
+    exit(0)
+
```
Comment (peri044) [inline tests/modules/hub.py]: can you merge this in an else block with already existing snippet at the top ? 
```py
if importlib.util.find_spec("torchvision"):
    import timm
    import torchvision.models as models
```

Comment (peri044) [reply tests/modules/hub.py]: also does `exit(0)` stop running this test and label it as failure ?

Comment (lanluo-nvidia) [reply tests/modules/hub.py]: good point, exit(0) will not mark the test as failure.
but reminded me that I should not completely exit(0) if torchvision is not installed.
Because we have some custom models located in the tests/modules/custom_models.py which should be downloaded via hub.py even without torchvision installed.

**Review by peri044 — APPROVED** [review]
LGTM
